MediaRecorder.html

```diff
@@ -353,13 +378,15 @@
             using the DOM manipulation task source, that runs the following
             steps:
             <ol>
-              <li>If <code>state</code> is "inactive" throw an
-              <code>InvalidStateError</code> DOMException and terminate these
-              steps. Otherwise:</li>
-              <li>Set <code>state</code> to "paused".</li>
-              <li>Stop gathering data into its current Blob (but keep the Blob
+              <li>If <a>state</a> is <a href="#dom-recordingstate-inactive">
+              inactive</a> throw an <code>InvalidStateError</code> DOMException
+              and terminate these steps. Otherwise:</li>
+              <li>Set <a>state</a> to <a href="#dom-recordingstate-paused">
+              paused</a>.</li>
```

Comment:
Same thing here, these two steps need to happen synchronously (throwing in a queued task is a no-op).